aw-client-rust/src/lib.rs

```diff
@@ -62,10 +62,10 @@ impl AwClient {
         &self,
         bucketname: &str,
         buckettype: &str,
-    ) -> Result<(), reqwest::Error> {
+    ) -> Result<Bucket, reqwest::Error> {
         let bucket = Bucket {
             bid: None,
-            id: bucketname.to_string(),
+            id: format!("{}_{}", bucketname, self.hostname),
```

Comment:
This is a breaking change. I suggest to keep the original behavior to avoid mishaps.

```suggestion
            id: bucketname.to_string(),
```

If you want your ID with a hostname suffix, better pass a different `bucketname` instead of adding this assumption to the `create_bucket_simple` method.